### DISCUSSION TITLE
The sound of a slider can stay in loop in the editor with certain timeline shifts

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#20814] [Q&A] [answered]
It's still unknown to me the exact scenario of why this happens, but as you can see from the videos below, for some reason sometimes the slider sound remains audible until there's a break / the end of the map / a time shift from the user.

https://user-images.githubusercontent.com/43073074/196527345-ac06e197-e2a0-4728-94d4-720291dc21d0.mp4

https://user-images.githubusercontent.com/43073074/196527368-ccfa53ee-8a57-480f-8b16-cf83bbba3271.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/20900
